docs/cheatsheet/cheatsheet.html

```diff
@@ -31,13 +31,14 @@
 <div class="header">
 <div class="usage">
 <span class="one">EarlGrey()</span>.selectElementWithMatcher(<span class="two">ViewMatcher</span>)
+                     .indexOf(<span class="three">index</span>)
```

Comment:
Hmm, I'd prefer to keep this section as a reference on how to create a canonical GREYElementInteraction. This can get rather unwieldy if we add more API's here.